src/core.js

```diff
@@ -69,6 +89,10 @@ export const present = L.lens(function present(history) {
   return S.nth(history.i, history.v)
 }, setPresentU)
 
+export const presentMut = L.lens(function present(history) {
+  return S.nth(history.i, history.v)
+}, setPresentUMut)
```

Comment:
```suggestion
export const presentReplace = L.lens(function presentReplace(history) {
  return S.nth(history.i, history.v)
}, setPresentReplaceU)
```

* The name in `function ...` can help with debugging (i.e. points to the correct function).